android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -369,4 +373,10 @@ public void onLayoutChange(View v, int left, int top, int right, int bottom, int
       mContentView.removeOnLayoutChangeListener(this);
     }
   }
-}
+
+  public void resumeZoomButtons()
```

Comment:
This function is not needed anymore, you can call `updateButtonsVisibility()` directly and it will handle the zoom buttons